libs/styles/styles.css

```diff
@@ -19,12 +19,12 @@
   /* Milo Color Specs */
   --color-white: var(--s2-white);
   --color-black: var(--s2-black);
-  --color-gray-100: var(--s2-gray-100);
-  --color-gray-200: #e8e8e8; /* milo only */
+  --color-gray-100: #f8f8f8;
```

Comment:
Is the color no longer available from Spectrum?

Reply:
Yes it is, there was a conflict in the designs I noticed when I was seeing the old footer background color using the `gray-100` s2 token color. It was a darker gray then specified in our mockups and the mockups called for this original color gray to still be used. I kept both this milo gray-100 and the s2 gray-100 in temporarily to see if it needed to be discussed with design, but that issue is actually fixed with correct usage of the global footer using the `--feds-background-footer--light`